designs/developer/4.6/HPA/index.md

```diff
@@ -0,0 +1,84 @@
+---
+parent: Developer
+version: 4.6
+---
+
+# Horizontal Pod Autoscaler
+
+## Description
+
+Horizontal pod autoscalers can be applied to workloads so that pods will scale up and down as needed so resources are not being wasting when waiting for peak resources. Horizontal pod autoscalers can only be applied to Deployment/Deployment Config workloads, not Knative services. A user can add, edit, and remove a horizontal pod autoscaler. 
+
+
+## Add a Horizontal Pod Autoscaler
+
+To add a horizontal pod autoscaler from a deployment or deployment config, the user clicks the Actions dropdown and selects 'Add Horizontal Pod Autoscaler'. 
+
+![Add horizontal pod autoscaler](img/1-hpa.jpg)
+
+### Add Horizontal Pod Autoscaler- Form
+
+The user can input Horizontal Pod Autoscaler attributes via a form or YAML editor. The form features limited field options so if the user wants to add custom metrics they should use the YAML editor. The info alert featured at the top of the form makes the user aware of this.
+
+![Add horizontal pod autoscaler form](img/2-hpa.jpg)
```

Comment:
For the min/max pods component I'm curious if this coming from PF. Wondering about the icons for them specifically, Or is this something that's already used in the console somewhere?

Reply:
It's currently used in other places in the console so that's where the min/max pods component came from.